addon/components/bs-form/element/control/switch.js

```diff
@@ -0,0 +1,20 @@
+import { action } from '@ember/object';
+import { tagName } from '@ember-decorators/component';
+import Control from '../control';
+import deprecateSubclassing from 'ember-bootstrap/utils/deprecate-subclassing';
+
+/**
+
+ @class FormElementControlSwitch
+ @namespace Components
+ @extends Components.FormElementControl
+ @private
+ */
+@tagName('')
```

Comment:
Hm, reminds me that we still have quite some work left over to move things to Glimmer components. It's ok to do this here as anything else would be out of scope for this PR, but still a bit unfortunate that we basically add additional tech debt here. Again, not your fault!